src/build.ts

```diff
@@ -51,6 +51,9 @@ export async function build(
       default: true,
     })) as PackageJson) || ({} as PackageJson);
 
+  // Prefer `publishConfig` when defined
+  Object.assign(pkg, pkg.publishConfig);
```

Comment:
We should at least avoid this when in stub mode (since it is for development) -- generally nice idea!